blackjack/api/views.py

```diff
@@ -20,8 +20,17 @@ def __init__(self):
         random.shuffle(self.deck_cards)
 
 
+@api_view(["GET", "POST", "PUT"])
+def deal(request, id):
+    data = json.loads(request.body)
+    username = data['username']
+    deck = data['deck']
+    player_hand = data['player_hand']
+    dealer_hand = data['dealer-hand']
 
+    # how do I get this into the hand that called the function?
+    card_dealt = Game.deck_cards.pop()
 
-@api_view(["GET"])
-def deal(request, id):
-    return JsonResponse(data={'card': 2}, status=status.HTTP_200_OK)
+    GameState.objects.create(username=username, deck=deck, player_hand=player_hand, dealer_hand=dealer_hand)
```

Comment:
you just dealt a card (`card_dealt`). how does that change the contents of the `deck` variable, and the `player_hand` variable?